src/pubsub/message/sign.js

```diff
@@ -49,6 +51,7 @@ async function verifySignature (message) {
   const pubKey = await messagePublicKey(message)
 
   // verify the base message
+  // @ts-ignore - may not have signature
```

Comment:
its safer to either make it required or protect with and if clause

Reply:
Cannot be required by multiple signing policies in the pubsub spec. Will add an if